datafusion/tests/sql/expr.rs

```diff
@@ -499,6 +499,16 @@ async fn test_interval_expressions() -> Result<()> {
     Ok(())
 }
 
+#[cfg(feature = "unicode_expressions")]
```

Comment:
I didnt find any helpers to assert that function return error, should I introduce a new one or did I miss something?

Reply:
I did not find one either. 

You could potentially use the `#[should_panic` annotation. For example, something like (untested):

```rust
#[tokio::test]
#[should_panic(expected = "negative substring length not allowed")]
async fn test_substring_expr_bad() -> Result<()> {
    test_expression!("substring('alphabet' from -1 for 1)", "l");
}
```




A helper would be great